grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -310,7 +310,7 @@ class OrderController {
                 try {
                     orderService.deleteOrder(orderInstance)
                     flash.message = "${warehouse.message(code: 'default.deleted.message', args: [warehouse.message(code: 'order.label', default: 'Order'), orderInstance.orderNumber])}"
-                    redirect(action: "list", params: params)
+                    redirect(controller: "order", action: "list", params: [orderType: Constants.PUTAWAY_ORDER, status: OrderStatus.PENDING])
```

Comment:
controller: "order" probably redundant? to be checked

Reply:
let me check it

Reply:
removed